### DISCUSSION TITLE
Model too high

### DISCUSSION THREAD

**Discussion by outandaboutar** [AR-js-org/AR.js#256] [General]
Hi
I have follow the guide and managed to get it working but with a couple issues I am hoping someone can advise on?  I am not experienced with AR at all so I dont know really what to expect or how to fix issues so any advice would be much appreciated.

The 3d models display but they are always raised too high above the ground, i have to point the camera upwards towards the sky, and the gps location range seems quite wide is there a way to shrink it or would providing a more precise gps location fix that?

Thank you!
Dan

**Comment by jonnypage** [comment]
On the `<a-entity>` or `<a-box>` or whatever you are using, you can set `position` and `scale`.

Something like this:

```
        <a-entity
          id="my-3d-model"
          gltf-model="/path/to/model.gltf"
          scale="1 1 1"
          position="50 0 -200"
        >
        </a-entity>
```
Position and scale are X Y Z

basically.. 
X is 'side to side'
Y is 'up and down'
Z is 'in and out' 

For more information you can check out the a-frame documentation on positioning:
https://aframe.io/docs/master/components/position.html

**Reply by outandaboutar** [reply]
Thank you, that worked!  Sorry for my late reply, I've been super busy with other things.  I dont know the limitations of this or the AR technology.  Is there a way to give the impression of getting closer to the object and raising its size?  Or to move around the object so that you can see the different sides?  I get the feeling AR is not as accurate as you would hope?  Least not when its based on GPS location?

**Reply by nickw1** [reply]
@outandaboutar GPS is accurate to a few metres at best, so it is not accurate enough to be able to move round an object and see all sides. However it is definitely accurate enough to get an impression of the objects becoming larger as you get close to them.

It is hoped in the future to use a combination of GPS and surface plane detection to improve the realism of object placement so that they actually appear as if they are 'planted' in the ground.

**Reply by outandaboutar** [reply]
I figured as much, thank you for the message!
